src/Tests/Microsoft.NET.Pack.Tests/GivenThatWeWantToPackAHelloWorldProject.cs

```diff
@@ -84,5 +84,54 @@ public void It_fails_if_nobuild_was_requested_but_build_was_invoked()
                 .And
                 .HaveStdOutContaining("NETSDK1085");
         }
+
+        [Fact]
+        public void It_packs_with_release_if_PackRelease_property_set()
+        {
+            var helloWorldAsset = _testAssetsManager
+               .CopyTestAsset("HelloWorld", "PackHelloWorld")
+               .WithSource();
+
+            System.IO.File.WriteAllText(helloWorldAsset.Path + "/Directory.Build.props", "<Project><PropertyGroup><PackRelease>true</PackRelease></PropertyGroup></Project>");
+
+            new BuildCommand(helloWorldAsset)
+               .Execute()
+               .Should()
+               .Pass();
+
+            var packCommand = new DotnetPackCommand(Log, helloWorldAsset.TestRoot);
+
+            packCommand
+                .Execute()
+                .Should()
+                .HaveStdOutContaining("Release")
```

Comment:
It might be better to check that the package is created in the `bin\Release` output folder.